examples/time-formatting-parsing/time-formatting-parsing.go

```diff
@@ -1,50 +1,54 @@
-// Go supports time formatting and parsing via
-// pattern-based layouts.
+// Go supporta la formattazione di struct di tipo `time`
```

Comment:
`time` in sé non è uno struct, `time` è un pacchetto, semmai possiamo dire di tipo `time.Time`. O altrimenti non diciamo nemmeno che è uno struct, dicendo la formattazione/formatting del tempo (devon ammettere però che suona un po' male)